HW31+/flask_app/views.py

```diff
@@ -23,12 +24,22 @@ def users():
                  "Алина", "Алиса", "Алла", "Алсу", "Альба", "Альбина", "Аля", "Амалия",
                  "Амина", "Анастасия", "Ангелина", "Анжела", "Анжелика", "Анисья",
                  "Анна", "Антонина", "Аполлинария", "Арина", "Астрид", "Белла", "Берта", "Валентина"]
-    if request.args:
-        lst2 = sample(lst_names, k=int(request.args.get('count')))
+    if session.get('user'):
+        if 'count' in request.args:
+            if request.args.get('count').isdigit():
+                lst2 = sample(lst_names, k=int(request.args.get('count')))
+            else:
+                return "Value 'count' should be a number"
```

Comment:
You can ignore this param if it is incorrect, it is not necessary to return an error here